labs/ipfs-naming/concepts.mdx

```diff
@@ -0,0 +1,62 @@
+---
+meta:
+  title: IPFS Naming - Concepts
+  description: This page explains all concepts related to IPFS Naming
+content:
+  h1: IPFS Naming - Concepts
+  paragraph: This page explains all concepts related to IPFS Naming
+tags: ipfs-naming naming ipfs versioning labs web3 cid
+categories:
+  - labs
+  - naming
+  - ipfs-naming
+---
+
+<Concept opened>
+
+## Content-based addressing
+
+Data over IPFS is identified by its content, not by its location (like HTTP does), providing a new way to access data through a unique content identifier (CID). Consequently, data is immutable and verifiable, allowing any device to be involved in the network, enhancing the redundancy.
+
+</Concept>
+
+<Concept opened>
+
+## Content Identifier (CID)
+
+The content identifier is string of character that points to a content. The content can be available on the network, or not. The CID is composed from the cryptographic hash of the content. It is used to identify and fetch the content through the network.
+
+</Concept>
+
+<Concept opened>
+
+## Distributed Hash Table (DHT)
+
+A distributed hash table (DHT) is a distributed system for mapping keys to values. In the context of the IPFS naming service, this is the association of a key (public hash key) with a record (value register in the IPFS naming service).
+
+</Concept>
+
+<Concept opened>
+
+## InterPlanetary File System (IPFS)
+
+IPFS is a peer-to-peer protocol, made by [Protocol Labs](https://protocol.ai/), for decentralized storage and content sharing.
+
+</Concept>
+
+
+<Concept>
+
+## Name
+
+The IPNS name corresponds to the cryptographic hash of your public key. It is linked to a value or record denoted by either `/ipfs/cid` or `/ipns/key`. Each name has a constant lifespan and is subject to regular updates by the service. Additionally, the record can be modified by the holder of the private key, ensuring continuous verifiability.
+
+</Concept>
+
+<Concept>
+
+## Peer-to-peer (P2P)
+
+Architecture that partitions tasks or workloads between peers. Peers are equally privileged participants in the network. They form a peer-to-peer network of nodes.
+
```

Comment:
Just a thought but maybe add a description what is a kubo client to the concepts?

Reply:
will be added later 🙏🏻